tests/v2/validation/provisioning/ssh.go

```diff
@@ -0,0 +1,26 @@
+package provisioning
+
+// create all tests here that use executable tests
```

Comment:
Nit; I think the comment here can be useful to have as a broad overview of what this file is. As such, maybe we can expand it a bit on this to be more specific as opposed to executable tests?